indra/resources/update_resources.py

```diff
@@ -6,6 +6,8 @@
 import rdflib
 import logging
 import requests
+from tqdm import tqdm
```

Comment:
While maintaining the christmas tree, I had to do a `from ...` import before a normal one. Is this okay? What's proper christmas tree etiquette?